src/main/java/uk/gov/hmcts/reform/civil/client/LocationReferenceDataApiClient.java

```diff
@@ -18,13 +18,6 @@ List<LocationRefData> getCourtVenueByName(
         @RequestParam("court_venue_name") final String courtVenueName
     );
 
-    @GetMapping(value = "/refdata/location/court-venues")
-    List<LocationRefData> getCourtVenueByEpimmsId(
```

Comment:
Are we 100% sure that this is not used by the front-end anywhere? Or any other service?
Maybe we should mark it deprecated rather than

Reply:
Its wrong- if FE using that need to be change as well.